app/controllers/application_controller.rb

```diff
@@ -1052,12 +1053,15 @@ def view_to_hash(view)
 
     root = {:head => [], :rows => []}
 
+    no_checkbox = @embedded || @no_checkboxes
+    no_listicon = %w(miqaeclass miqaeinstance).include?(view.db.downcase)  # do not add listicon for AE class show_list
+
     # Show checkbox or placeholder column
-    unless @embedded || @no_checkboxes
+    unless no_checkbox
```

Comment:
Please don't use negative variable names, and then a negative clause.  double+negatives are so confusing.  Prefer `has_checkbox` and `if has_checkbox`